src/applications/representative-appoint/containers/ConfirmationPage.jsx

```diff
@@ -17,8 +115,22 @@ export default function ConfirmationPage({ router }) {
 
       if (signedFormError) setSignedFormError(false);
     },
-    onClickContinueButton: () => {
+    onClickContinueButton: async () => {
       if (signedForm) {
+        try {
+          await sendNextStepsEmail({
+            emailAddress,
+            firstName,
+            representativeType: getRepType(),
+            representativeName,
+            representativeAddress,
+            formNumber: getFormNumber(),
+            formName: getFormName(),
+          });
+        } catch (error) {
+          // Should we set an error state to display a message in the UI?
```

Comment:
We need to tell the user on the `/next-steps` page that the call to send the email failed.